wordpress-mobile/WordPress-iOS
# Show meaningful error when xmlrpc broken

Fixes #10293 

NOTE: This uses an updated `WordPressKit` pod. I will update the `Podfile` to a valid version after it's been approved.

Related `WordPressKit` PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/48


To test:

---
- On the login page, select `Log in by entering your site address.`
- Enter an address that is _not_ a WordPress site.
- Verify the message is now `Couldn't connect to the WordPress site. There is no valid WordPress site at this address. Check the site address (URL) you entered.`

![not_wp_site](https://user-images.githubusercontent.com/1816888/47391844-23667f80-d6d8-11e8-8e5c-2c89e7d8fd88.png)

---
- On the login page, select `Log in by entering your site address.`
- Enter an address of a WP site that has RSD set up but XML-RPC is not.
  - Feel free to ping me or @SiobhyB for an example customer site if needed.
- Verify the message is now `Couldn't connect. Required XML-RPC methods are missing on the server.`

![xmlrpc_broken](https://user-images.githubusercontent.com/1816888/47391973-73454680-d6d8-11e8-9a76-17cd2077fbdf.png)
